docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -104,53 +136,13 @@ spec:
           alpha.kubernetes.io/nvidia-gpu: 2
 ```
 
-This will ensure that the pod will be scheduled to a node that has a `Tesla K80` or a `Tesla P100` Nvidia GPU.
-
-### Warning
-
-The API presented here **will change** in an upcoming release to better support GPUs, and hardware accelerators in general, in Kubernetes.
-
-## Access to CUDA libraries
-
-As of now, CUDA libraries are expected to be pre-installed on the nodes.
-
-To mitigate this, you can copy the libraries to a more permissive folder in ``/var/lib/`` or change the permissions directly. (Future releases will automatically perform this operation)
-
-Pods can access the libraries using `hostPath` volumes.
-
-```yaml
-kind: Pod
-apiVersion: v1
-metadata:
-  name: gpu-pod
-spec:
-  containers:
-  - name: gpu-container-1
-    image: gcr.io/google_containers/pause:2.0
-    resources:
-      limits:
-        alpha.kubernetes.io/nvidia-gpu: 1
-    volumeMounts:
-    - mountPath: /usr/local/nvidia/bin
-      name: bin
-    - mountPath: /usr/lib/nvidia
-      name: lib
-  volumes:
-  - hostPath:
-      path: /usr/lib/nvidia-375/bin
-    name: bin
-  - hostPath:
-      path: /usr/lib/nvidia-375
-    name: lib
-```
+This will ensure that the pod will be scheduled to a node that has a `Tesla K80` or a `Tesla P100` NVIDIA GPU.
 
 ## Future
 
-- Support for hardware accelerators is in its early stages in Kubernetes.
-- GPUs and other accelerators will soon be a native compute resource across the system.
```

Comment:
Why we delete this?

Reply:
> -- GPUs and other accelerators will soon be a native compute resource across the system.
 
That's not the case anymore

> -- Support for hardware accelerators is in its early stages in Kubernetes.

I can add this back. My idea was that we're planning to move to beta for this so not really early stage anymore.

Reply:
Nevermind, the second line wasn't removed, I edited it to:
> Support for hardware accelerators is in its early stages in Kubernetes but is expected to graduate to beta in 1.10.

Reply:
lgtm